### DISCUSSION TITLE
Content Part Handler

### DISCUSSION THREAD

**Discussion by EricBui619** [OrchardCMS/OrchardCore#9185] [Q&A]
My scenario is to have a content part of a content type. So I create a content part handler for every time I publish or update a content type that has that part, handler will trigger to run my some code. Unfortunately, my handler doesn't work. Do I have to add a driver before handler in startup?
Btw, I wanna call api from SPA to update content part, it's possible to handler triggered?

![image](https://user-images.githubusercontent.com/30250345/114965041-0b7fa900-9e9a-11eb-8c83-fdf70ca16b4a.png)

![image](https://user-images.githubusercontent.com/30250345/114965068-176b6b00-9e9a-11eb-939f-a3e8cc171bc5.png)

![image](https://user-images.githubusercontent.com/30250345/114965105-26eab400-9e9a-11eb-812d-7d391ee46b6b.png)

**Comment by hishamco** [comment]
Have a look to `TitlePartHandler` as example https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Title/Handlers/TitlePartHandler.cs
